docs/index.md

```diff
@@ -9,7 +9,7 @@ more.
 
 [Installation](docs/getting-started/)
 
-## Example usage
+**Example usage**
```

Comment:
That is because it is not part of a category https://github.com/sequelize/sequelize/pull/3165/files#diff-45ee620855dad847bea76b49932ad7dbL14